### DISCUSSION TITLE
Select left or right channel for mono audio input.

### DISCUSSION THREAD

**Discussion by 777paranoia** [bangcorrupt/hacktribe#176] [Ideas]
hey, so not really feeling like this would be mechanically possible but thought i'd throw it out there. always dreamed of having an e2s with multiple assignable outputs and thought i'd suggest finding a way to repurpose the headphone output buss as a separate output channel (like sr-16s' main / aux outputs) which the voices could be assigned to.

also, if there's any way to split the stereo audio input into two individual osc options as separate mono channels that can each be assigned to different pads, that'd be kinda sick as well, and seems like it could be somewhat easy to execute. 

if that doesnt make sense in writing; what i mean is the current options are "Audio In St" and "Audio In Mn", new options would be "Audio In St", "Audio In St Left/Mn", "Audio In St Right/Mn", and Audio In Mn"

thanks

**Comment by bangcorrupt** [comment]
> repurpose the headphone output

There is only one DAC feeding both the main and headphone outputs, so this is not possible.  With some hardware hacking it may be possible to use a multichannel DAC with the DSP  outputting a TDM format.  

> stereo audio input into two individual osc options as separate mono channels

This might be possible.
